tests/e2e/run-scanner-v4.bats

```diff
@@ -306,6 +315,18 @@ verify_scannerV4_matcher_deployed() {
     wait_for_ready_pods "${namespace}" "scanner-v4-matcher" 120
 }
 
+verify_central_scannerV4_env_var_set() {
+    local namespace=${1:-stackrox}
+
+    local central_env_vars="$("${ORCH_CMD}" -n "${namespace}" get deploy/central -o jsonpath="{.spec.template.spec.containers[?(@.name=='central')].env}")"
+    local scanner_v4_enabled="$(echo $central_env_vars | jq 'map(select(.name == "ROX_SCANNER_V4" and .value == "true")) | length > 0')"
```

Comment:
Would feel a bit straightforward to me to extract the value for `ROX_SCANNER_V4` and then compare it to `"true"`, instead of extracting all entries with `ROX_SCANNER_V4=true`, compute the length of it and check if that length is `> 0` and then compare the result of that comparison with `"true"`.